### ISSUE TITLE
Add +/- mouseover highlight

### ISSUE THREAD

**Issue by DanRathbun** [MSP-Greg/MSP-Greg.github.io#2]
> Dan - added the +/- highlight to my doc site...

**It looks good / works well !** 

Hey here's an issue for you to close... 

;)

**Comment by MSP-Greg** [comment]
@DanRathbun 

Sorry for the delay, I didn't realize that one needed to watch one's own (non-forked) repos.

I've been bouncing between RubyInstaller & RubyInstaller2 & msys2/mingw, adding more DSL functionality to doc'ing Rails, and trying to integrate YARD RSpec tests into YardT2 (my plugin, 'T2').

Once I get the the tests where I want them, I'll release T2 as a gem.

In the meantime, I may ask for your help with some T2 issues re font size.  I've started a web page that  gives me all the font metrics, I need to finish it and add it to the site, then (hopefully) get some feedback as to how the site is rendering on various devices.

I may just add a 'settings' button to the UI, which could include font size +/-.  I really don't want to, as the UI is pretty crowded on a tablet.  Maybe I'll stack the buttons in the upper right corner...

BTW, re having Ruby docs on the SU/Trimble site, I've got code that runs YARD/T2, and is capable of pulling a git repo to any branch or tag.  I could mod it to allow pulling to a specific tag, which would allow doc'ing to a specific patch of Ruby.  Specific code for Ruby, Rails, & AWS, and a base module.

**Comment by DanRathbun** [comment]
> Once I get the the tests where I want them, I'll release T2 as a gem.

I cannot find anywhere what "T2" means?  "Take 2"? "Terminator 2"?

**Comment by MSP-Greg** [comment]
@DanRathbun Sorry, kind of running out to lunch when I wrote that.

By YARD convention, YARD plugin gems should be named yard-*.

So, way back I named my gem yard-t2, with (again, somewhat by convention) a class name of YardT2.  I expected it to be just a replacement template system that updated the UI to work with tablets.

So 'T2' is shorthand for the plugin gem that I'm using to create the .io doc site.  Obviously, it's gone a bit past that...
